pkg/core/interop/contract/call.go

```diff
@@ -69,6 +69,12 @@ func Call(ic *interop.Context) error {
 		return fmt.Errorf("method not found: %s/%d", method, len(args))
 	}
 	hasReturn := md.ReturnType != smartcontract.VoidType
+
+	ic.InvocationCalls = append(ic.InvocationCalls, state.ContractInvocation{
+		Hash:   u,
+		Method: method,
+		Params: stackitem.NewArray(args),
```

Comment:
Another question is: should we track invocations for native contracts? Current way of tracking doesn't consider native invocations, although for Notifications system all contracts are taken into account:
https://github.com/nspcc-dev/neo-go/blob/d47fe392fb144cc3662b0e5770e253f8d5207f27/pkg/core/native/interop.go#L15-L16
I consider that these two systems should be aligned (or Invocations system should be aligned with VM InvocationsTree and also include all calls), so that users may rely on some unification in behaviour. Also, it might be useful for users to be aware of native calls.

Reply:
I agree calls to native contracts should be included. I do think we should not include calls made with the `OnPersist` and `PostPersist` triggers as these are just system events. I'm afraid those will blow up the storage size considering they happen every block (e.g. gas minting) and I currently see no real value in that information. Would you agree? 

Luckily the trigger type is part of the interop context so it should be easy to filter.

Reply:
> blow up the storage size

Absolutely.

> I currently see no real value in that information

Well, given the fact that we have proper notifications for token transfers and application logs for OnPersist/PostPersist, some additional Invocations information seems to be redundant, indeed. It's not allowed to call OnNEP17Payment for Neo/Gas OnPersist/PostPersist, and other native contracts also do not have nested calls as far.

So I agree, there's no need to track native system invocations.